projects/angular/src/lib/maskito.directive.ts

```diff
@@ -55,6 +55,7 @@ export class MaskitoDirective implements OnDestroy, OnChanges {
         }
 
         ngZone.runOutsideAngular(() => {
+            this.maskedElement?.destroy();
```

Comment:
@TheHomeGera can you add unit test for catch this?

Reply:
sorry, I can't to write unit test on it, maybe cypress can catch it. 

The picture shows an example of how to catch. If the second options flies into ngOnChanges before the callback from runOutsideAngular is executed. Then destroy will not work, since destroy has worked, and there will be a double field setting already in the callback.
![image](https://github.com/user-attachments/assets/23df431c-4367-467f-96f9-a9cd4d3a02f6)

Reply:
`maybe cypress can catch it.` could you add?

Reply:
@TheHomeGera Hello!

I've written some Cypress tests for this case:
* https://github.com/taiga-family/maskito/pull/1630

Just rebase your branch and enable them in your PR.

You need to remove `.skip` (and `TODO` comments) for these tests:
https://github.com/taiga-family/maskito/blob/92a8f3208288e01648757c7c2e5f56867d404100/projects/demo-integrations/src/tests/component-testing/angular-predicate/angular-predicate.cy.ts#L178-L179

https://github.com/taiga-family/maskito/blob/92a8f3208288e01648757c7c2e5f56867d404100/projects/demo-integrations/src/tests/component-testing/angular-predicate/angular-predicate.cy.ts#L184-L185

Reply:
+